pkg/queue/manager_test.go

```diff
@@ -414,7 +414,7 @@ func TestRequeueWorkload(t *testing.T) {
 				_ = manager.AddOrUpdateWorkload(tc.workload)
 			}
 			info := workload.NewInfo(tc.workload)
-			if requeued := manager.RequeueWorkload(ctx, info); requeued != tc.wantRequeued {
```

Comment:
can you rename the test to `TestRequeueWorkloadStrictFIFO`? do we have one for BestEffort?